aten/src/ATen/cuda/CUDAEvent.h

```diff
@@ -45,6 +46,10 @@ struct TORCH_CUDA_CPP_API CUDAEvent {
     try {
       if (is_created_) {
         CUDAGuard guard(device_index_);
+        const auto* interp = c10::impl::CUDATraceTLS::get_trace();
```

Comment:
Nit: personally I tend to use `auto` very conservatively, because I think that hiding the actual type of a variable from the code has more downsides than benefits (i.e., it makes it faster to write the code, but it slows down reading it).

I tend to use auto only when it's really necessary (i.e., when it behaves like a template argument which can have different types in different cases), or when the type is specified immediately nearby, such as in `auto ptr = std::make_unique<SomeType>()`.

Reply:
You're right, fixed.